rosetta-source/src/main/rosetta/product-asset-type.rosetta

```diff
@@ -95,6 +95,7 @@ type InterestRatePayout extends PayoutBase: <" A class to specify all of the ter
 	bondReference BondReference (0..1) <"Reference to a bond underlier to represent an asset swap or Condition Precedent Bond.">
 	fixedAmount calculation (0..1) <"Fixed Amount Calculation">
 	floatingAmount calculation (0..1) <"Floating Amount Calculation">
+    spreadCalculationMethod SpreadCalculationMethodEnum (0..1) <"Method by which asset swap spread is calculated, par-par or proceeds.">
```

Comment:
Consider description as follows:
<"Method by which a spread is calculated. E.g., "par-par" or "proceeds" are methods by which an asset swap spread is calculated typically.">

Reply:
I modified the description to be more generic. Thanks for the suggestion!